### ISSUE TITLE
TypeError: Cannot read properties of null (reading 'includes')

### ISSUE THREAD

**Issue by donalffons** [sanity-io/sanity-plugin-mux-input#291]
I'm getting the above error message when using this package. The problem seems to be coming from the `upchunk` dependency. See [this issue](https://github.com/muxinc/upchunk/issues/111) for more information and a [suggestion for a fix](https://github.com/muxinc/upchunk/issues/111#issuecomment-1563057481), that worked for me.

**Comment by donalffons** [comment]
...in addition, I have to specify `experimental.esmExternals = "loose"` in my `next.config.js` as suggested [here](https://github.com/sanity-io/sanity-plugin-mux-input/issues/252#issuecomment-1557264695).

**Comment by hdoro** [comment]
@donalffons can you confirm that this is still an issue with the latest plugin & Sanity versions?

**Comment by donalffons** [comment]
No, not an issue any more, since the pr was merged. Just forgot to close this issue. Cheers!
